samples/openthread/cli/sample.yaml

```diff
@@ -22,7 +23,7 @@ tests:
       - nrf21540dk_nrf52840
   samples.openthread.cli.usb:
     build_only: true
-    platform_allow: nrf5340dk_nrf5340_cpuapp nrf52840dk_nrf52840 nrf52840dongle_nrf52840 nrf52833dk_nrf52833 nrf21540dk_nrf52840
+    platform_allow: nrf5340dk_nrf5340_cpuapp nrf5340dk_nrf5340_cpuapp_ns nrf52840dk_nrf52840 nrf52840dongle_nrf52840 nrf52833dk_nrf52833 nrf21540dk_nrf52840
     tags: ci_build
     extra_args: OVERLAY_CONFIG=overlay-usb.conf
     integration_platforms:
```

Comment:
Is it a mistake or this section does not need to be updated (I do not remember what this section does :( )

Reply:
I enabled the build target for all configurations.